.changeset/fix-modalEscape.md

```diff
@@ -0,0 +1,5 @@
+---
+'react-magma-dom': patch
+---
+
+fix(Modal): Added support for closing components that have menus with the escape key inside of modals while retaining an open modal state.
```

Comment:
Can we list all the components this affects? This is actually not a bug on the modal but on the individual components. Our adopters will want to know what components were changed in each version in case they run into problems.